connectors/google-pubsub-sink/v2.8.1.29/google-pubsub-sink.md

```diff
@@ -0,0 +1,377 @@
+>
+> - projects.topics.create
+> - projects.topics.get
+> - projects.topics.publish
+>
+> For more information about Google Cloud Pub/Sub API permissions, see [Google Cloud Pub/Sub API permissions: Access control](https://cloud.google.com/pubsub/docs/access-control).
+
+::: tabs
+
+@@@ Function Worker
+
+You can create a configuration file (JSON or YAML) to set the properties if you use [Pulsar Function Worker](https://pulsar.apache.org/docs/en/functions-worker/) to run connectors in a cluster.
+
+**Example**
+
+* JSON
+
+   ```json
+    {
+        "tenant": "public",
+        "namespace": "default",
+        "name": "google-pubsub-sink",
+        "inputs": [
+          "test-google-pubsub-pulsar"
+        ],
+        "archive": "connectors/pulsar-io-google-pubsub-{{connector:version}}.nar",
+        "parallelism": 1,
+        "configs": {
+          "pubsubCredential": "SECRETS",
+          "pubsubProjectId": "pulsar-io-google-pubsub",
+          "pubsubTopicId": "test-google-pubsub-sink"
+      }
+    }
+    ```
+
+* YAML
+
+    ```yaml
+    tenant: public
+    namespace: default
+    name: google-pubsub-sink
+    inputs:
+      - test-google-pubsub-pulsar
+    archive: connectors/pulsar-io-google-pubsub-{{connector:version}}.nar
+    parallelism: 1
+    configs:
+      pubsubCredential: 'SECRETS'
+      pubsubProjectId: pulsar-io-google-pubsub
+      pubsubTopicId: test-google-pubsub-sink
+    ```
+
+@@@
+
+@@@ Function Mesh
+
+You can create a [CustomResourceDefinitions (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to create a Google Cloud Pub/Sub sink connector. Using CRD makes Function Mesh naturally integrate with the Kubernetes ecosystem. For more information about Pulsar sink CRD configurations, see [sink CRD configurations](https://functionmesh.io/docs/connectors/io-crd-config/sink-crd-config).
+
+You can define a CRD file (YAML) to set the properties as below.
+
+```yaml
+apiVersion: compute.functionmesh.io/v1alpha1
+kind: Sink
+metadata:
+  name: google-pubsub-sink-sample
+spec:
+  image: streamnative/pulsar-io-google-pubsub:{{connector:version}}
+  className: org.apache.pulsar.ecosystem.io.pubsub.PubsubSink
+  replicas: 1
+  maxReplicas: 1
+  input:
+    topics: 
+      - persistent://public/default/destination
+  sinkConfig:
+    pubsubCredential: 'SECRETS'
+    pubsubProjectId: pulsar-io-google-pubsub
+    pubsubTopicId: test-google-pubsub-sink
+  pulsar:
+    pulsarConfig: "test-pulsar-sink-config"
+  resources:
+    limits:
+    cpu: "0.2"
+    memory: 1.1G
+    requests:
+    cpu: "0.1"
+    memory: 1G
+  java:
+    jar: connectors/pulsar-io-google-pubsub-{{connector:version}}.nar
+  clusterName: test-pulsar
+  autoAck: true
+```
+
+@@@
+
+:::
+
+# How to use
+
+You can use the Google Cloud Pub/Sub sink connector with Function Worker or Function Mesh.
+
+## Work with Function Worker
```

Comment:
consider using switch tabs to keep consistent overall?

::: tabs

@@@ Function Worker
@@@ Function Mesh

Reply:
For the `how to use` section. I tried to use multi-tab format. but it does not displayed in a reasonable way. Therefore, for the How to use section, we can just use sub-section.